lib/src/generic_dropdown.dart

```diff
@@ -132,7 +132,7 @@ class GenericDropdown extends StatefulWidget {
   State<GenericDropdown> createState() => _GenericDropdownState();
 }
 
-class _GenericDropdownState extends State<GenericDropdown> {
+final class _GenericDropdownState extends State<GenericDropdown> {
```

Comment:
here it is fine, because its a private class. however not really required since the private class is not exported anyway.